Sources/JavaScriptKit/FundamentalObjects/JSClosure.swift

```diff
@@ -234,3 +235,20 @@ func _free_host_function_impl(_ hostFuncRef: JavaScriptHostFuncRef) {
     JSClosure.sharedClosures[hostFuncRef] = nil
 }
 #endif
+
+#if hasFeature(Embedded)
+// cdecls currently don't work in embedded, and expose for wasm only works >=6.0
```

Comment:
There is no reason using `cdecl` instead of `silgen_name` here, so I think we can use `silgen_name` and call those functions with Swift calling convention. I'll address this after merging your change, so don't care about this comment, just a note for me.